packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -369,6 +371,7 @@ class BottomSheetCell extends Component {
 				onPress={ onCellPress }
 				onLongPress={ onLongPress }
 				style={ [ styles.clipToBounds, style ] }
+				testID={ 'cell-touchable' }
```

Comment:
I see this component already defines an accessibility label so you could use it for querying it instead of adding a test ID. The React Native testing library provides queries `ByA11yLabel` for this case ([reference](https://callstack.github.io/react-native-testing-library/docs/api-queries#bya11ylabel-byaccessibilitylabel-bylabeltext)).

Reply:
This was the one area where I thought having a dedicated testID might be advantageous, as this component's a11yLabel is really conditional. But, for the clear button tests we're trying to scope, I can also see how using the accessibility label might be more realistic here, so I will consider that as part of updating the tests.

Reply:
For the test case, one thing you could try is to pass a value through in the `accessibilityLabel` prop when rendering the `BottomSheetCell` component, this way you could use that value for querying it.

Reply:
Ah, yep! I did not even consider that it would be just as easy to pass the accessibilityLabel as a prop here for testing. Thanks for this suggestion!